Tests/test_Pathway.py

```diff
@@ -50,11 +49,17 @@ def test_Nodes(self):
     def test_Edges(self):
         a = Graph(["a", "b", "c", "d"])
         a.add_edge("a", "b", "label1")
-        self.assertEqual(a.child_edges("a"), [("b", "label1")])  # , "incorrect child edges")
+        self.assertEqual(
+            a.child_edges("a"), [("b", "label1")]
+        )  # , "incorrect child edges")
```

Comment:
I think we can remove this comment - which should let this fit on one line.

(Same for the two following cases too)